client/me/purchases/cancel-purchase/button.jsx

```diff
@@ -263,7 +263,7 @@ class CancelPurchaseButton extends Component {
 			}
 
 			if ( isSubscription( purchase ) ) {
-				text = translate( 'Cancel Subscription' );
+				text = translate( 'Disable Auto-renew' );
```

Comment:
This change is incorrectly showing up for refundable subscriptions also, when cancelling the subscription actually removes and refunds it right away (rather than disabling auto-renew):

![cancelling-refundable-purchase](https://user-images.githubusercontent.com/235183/126825205-23fd75e1-965a-405a-aebd-7b97a9ab665a.png)